File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
number = list(map(int, input().split()))
number.sort()

count = 0
answer = []

for i in range(3):
    for j in range(3):
        if number[i] == number[j]:
            count += 1 
    answer.append(count)
    count = 0

if max(answer) == 3:
    print(10000 + (number[answer.index(max(answer))] * 1000))
elif max(answer) == 2:
    print(1000 + (number[answer.index(max(answer))] * 100))
else:
    print((max(number) * 100))

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let [one, two, three] = input[0].split(" ").map(Number);

if(one === two && one === three && two === three){
  console.log(10000 + one * 1000);
}
else if(one !== two && one !== three && two !== three){
  console.log(Math.max(one, two, three) * 100);
}
else{
  if(one === two){
    console.log(1000 + one * 100);
  }
  else if(one === three){
    console.log(1000 + one * 100);
  }
  else if(two === three){
    console.log(1000 + two * 100);
  }
}